src/site/_utils/get-image-path.js

```diff
@@ -20,13 +20,14 @@ const site = require("../_data/site");
 /**
  * Takes a path to an image and converts it to an image CDN path if we're in
  * a production environment.
- * @param {!string} src A path to an image asset.
+ * @param {!string} src A path to an image asset.\
  * @param {!string} pageUrl The url for the current page.
  * @return {string} An image path. May be converted to an image CDN path if
  * it's a production environment.
  */
 module.exports = function getImagePath(src, pageUrl) {
   let imagePath = path.join(pageUrl, src);
+
```

Comment:
i think if you remove this newline then this file won't have any changes and you can remove it from the PR